quantum/keyboard.c

```diff
@@ -111,15 +111,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 static uint32_t last_input_modification_time = 0;
 uint32_t        last_input_activity_time(void) {
-    return last_input_modification_time;
+           return last_input_modification_time;
```

Comment:
```suggestion
    return last_input_modification_time;
```
Apparently your version of `clang-format` does some things differently, but you need to match the one in the `qmkfm/qmk_cli` image to please the CI.

Reply:
saw that. beyond irritating